.github/workflows/main.yml

```diff
@@ -14,20 +14,20 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/setup-go@v5
+      - uses: actions/setup-go@v6
```

Comment:
This is fine, I'm happy to approve - however some projects have been switching to use hashes to be me more secure against possible future malicious versions:
https://docs.github.com/en/actions/reference/security/secure-use#using-third-party-actions

Since you're touching a lot of these now it might be worth adopting that practice too, but I won't insist on it at this time.